librina/include/librina/cdap_rib_structures.h

```diff
@@ -48,43 +48,17 @@ typedef struct auth_policy {
 	SerializedObject options;
 } auth_policy_t;
 
-typedef struct destination_info {
-	/// DestinationApplication-Entity-Instance-Id (string), optional, not validated by CDAP.
-	/// Specific instance of the Application Entity that the source application
-	/// wishes to connect to in the destination application.
+typedef struct ep_info {
+	/// Application-Entity-Instance-Id (string), optional, not validated by CDAP.
 	std::string ae_inst_;
-	/// DestinationApplication-Entity-Name (string), mandatory (optional for the response).
-	/// Name of the Application Entity that the source application wishes
-	/// to connect to in the destination application.
+	/// Application-Entity-Name (string), mandatory (optional for the response).
 	std::string ae_name_;
-	/// DestinationApplication-Process-Instance-Id (string), optional, not validated by CDAP.
-	/// Name of the Application Process Instance that the source wishes to
-	/// connect to a the destination.
+	/// Application-Process-Instance-Id (string), optional, not validated by CDAP.
 	std::string ap_inst_;
-	/// DestinationApplication-Process-Name (string), mandatory (optional for the response).
-	/// Name of the application process that the source application wishes to connect to
-	/// in the destination application
+	/// Application-Process-Name (string), mandatory (optional for the response).
 	std::string ap_name_;
-} dest_info_t;
+} ep_info_t;
```

Comment:
Agree on this change, removing duplicates.

What is "ep_" standing for? A documentation before `typedef struct ep_info` (and before all types), would be nice so that doxygen can capture it.